android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -622,8 +622,8 @@
 		"startBias": ["Plains"],
 		"preferredVictoryType": "Domination",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O great Temuujin, universal leader of the mighty Mongol Empire! Your name alone was enough to make cities crumble, and the sound of the mighty Mongol cavalry to make flee your enemies. O Great Khan! Once Northern Asia united into a mighty people, producing one of the greatest army the world has ever known, your skillfull mounted archers conquered in only a few years, most of China, Eastern Asia, and even reached the gates of Europe at the West. With cunning diplomacy and brilliant warfare, you swept your enemies one after another, creating the greatest empire ever seen, reducing Roman or Greek conquests to minor achievements.",
```

Comment:
To make your enemies flee.
One of the greatest armies the world has even known
Skillful